README.md

```diff
@@ -78,6 +79,21 @@ To launch the interactive CLI, run the executable without any arguments.
 
 Run `ficsit help` to see a list of available commands.
 
+## Game Installations
+
+The app needs to know where the game is installed. On Steam, this will be
+
+    C:\Program Files (x86)\Steam\steamapps\common\Satisfactory
```

Comment:
Not always. Steam installations should be auto-detected anyway

Reply:
Not on MacOS they aren't

Reply:
> Not always. Steam installations should be auto-detected anyway

This is ficsit-cli, not SMM. Only SMM auto-detects installs